src/app/child-dev-project/attendance/add-day-attendance/roll-call/roll-call.component.html

```diff
@@ -46,55 +64,3 @@
     Attendance completed.
```

Comment:
It was not clear to me that I have to press this area in order to save and complete the attendance as it looks like the attendance is **already** completed. 

Maybe call this something like _Complete attendance and close roll call_

Reply:
added button for clearer action